### PR TITLE
Fastem overlap from conf settings and fix overlap for small ROA size

### PR DESCRIPTION
**Pull request by tepals**

### CODE REVIEW

File: src/odemis/acq/fastem.py [hunk 1]
```diff
@@ -147,12 +147,24 @@ def _calculate_field_indices(self):
         field_size = (field_res[0] * px_size[0],
                       field_res[1] * px_size[1])
 
-        # Note: floating point errors here, can result in an additional row or column of fields (that's fine)
+        # Note: Megafields get asymmetrically extended towards the right and bottom.
```
Comment (pieleric) [inline src/odemis/acq/fastem.py:150]: For info, the code of tiledacq does extend symmetrically. If you wanted to improve the code here, you're welcome to steal that code ;-)

**Review by pieleric — CHANGES_REQUESTED** [review]

File: src/odemis/gui/model/__init__.py [hunk 2]
```diff
@@ -1263,7 +1263,11 @@ class FastEMAcquisitionGUIData(MicroscopyGUIData):
     def __init__(self, main):
         assert main.microscope is not None
         super(FastEMAcquisitionGUIData, self).__init__(main)
-
+        _conf = get_general_conf()
+        # Get the overlap from /home/[user]/.config/odemis/odemis.config, it should be defined as follows:
+        # [acquisition]
+        # overlap = 0.04
+        self.overlap = float(_conf.config.get("acquisition", "overlap", fallback=0.0))
```
[the remark this answers is not in the document]
Comment (tepals) [reply src/odemis/gui/model/__init__.py]: Done, can you do another review? I'll squash the first and last commit once afterwards.

**Review by pieleric — APPROVED** [review]
Be aware there is now a conflict with master.

File: src/odemis/gui/conf/file.py [hunk 3]
```diff
@@ -541,4 +550,3 @@ def get_sh_calib(self, shid):
                 logging.exception("Failed to read calibration data")
 
         return None
-
```
Comment (Sabrina85) [inline src/odemis/gui/conf/file.py:544]: Pep8 is one empty line at the end of a file or not?

File: src/odemis/gui/conf/file.py [hunk 4]
```diff
@@ -248,6 +248,7 @@ def __init__(self):
         # is completed.
         self.default.set("acquisition", "fn_ptn", u"{datelng}-{timelng}")
         self.default.set("acquisition", "fn_count", "0")
+        self.default.set("acquisition", "overlap", "0.0")
```
Comment (Sabrina85) [inline src/odemis/gui/conf/file.py:251]: I don't know how the concept works, so maybe a stupid question. Does this also work for any other type of microscope?

**Review by Sabrina85 — CHANGES_REQUESTED** [review]
Looks good to me. You could update the date in the header.

File: src/odemis/gui/conf/file.py (hunk 3 above)
Comment (tepals) [reply src/odemis/gui/conf/file.py:544]: yes, there were two in this file, github sort of hides the last empty line.

File: src/odemis/gui/conf/file.py (hunk 4 above)
Comment (tepals) [reply src/odemis/gui/conf/file.py:251]: yes, you could now use an overlap constant in any microscope, which you can access with `acqui_conf = conf.get_acqui_conf()`

File: src/odemis/gui/conf/file.py (hunk 3 above)
Comment (Sabrina85) [reply src/odemis/gui/conf/file.py:544]: Ah didn't know that it is hidden as I could not expand the file further.

File: src/odemis/gui/conf/file.py (hunk 4 above)
Comment (Sabrina85) [reply src/odemis/gui/conf/file.py:251]: Thanks - that is nice! And if you don't want an overlap it also works with any microscope right?

**Review by Sabrina85 — APPROVED** [review]

File: src/odemis/gui/conf/file.py (hunk 4 above)
Comment (tepals) [reply src/odemis/gui/conf/file.py:251]: yes

**Pull request merged**